File: DSA LeetCode/1960-check-if-the-sentence-is-pangram/solution.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    const set = new Set();
    for (const s of sentence) {
        set.add(s);
    }
    console.log(set.size)
    return set.size === 26 ? true : false;
};

File: DSA LeetCode/1960-check-if-the-sentence-is-pangram/solution.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        check_set = set()
        for index in range(len(sentence)):
            if (sentence[index] not in check_set):
                check_set.add(sentence[index])
                
        if (len(check_set) == 26):
            return True
        return False
